src/core_plugins/region_map/public/choropleth_layer.js

```diff
@@ -119,6 +122,10 @@ export default class ChoroplethLayer extends KibanaMapLayer {
       } catch (e) {
         this._loaded = true;
         this._error = true;
+        this._notifier.error(`Could not load vector layer at ${geojsonUrl}. 
```

Comment:
Can't we distinguish here between a file not found (which should result in the response code 404) and a CORS error (which shouldn't result in any response code) or in general any error? I think if we can distinguish at least 404 not found from all other errors, the usefulness of this error message could massively improved, by splitting it up into two different error messages, since we wouldn't need to confuse users with CORS error messages, that are always very technical and hard to get, if we know the file actually just doesn't exist.

In general the vector layer can only be configured via `kibana.yml`? Does it make sense at all to show a notification to the user using it, since they cannot do anything against it, rather then just logging something to console and/or server logs?

Reply:
^ Notify is already a toast-error, but I agree we can try and use that new component. Not 100% sure if we can detect CORS-issue, there's some security issues related to that, but will look into it.